pkg/routers/api_v1_test.go

```diff
@@ -72,6 +72,7 @@ func (suite *APIV1TestSuite) SetupTest() {
 		middleware:      &MockOptlyMiddleware{},
 		handlers:        MockHandlers{},
 		metricsRegistry: metricsRegistry,
+		enableOverrides: true,
```

Comment:
I think we need a new test in this file, covering changes to `routers/api_v1.go`.